modules/clientmanagements/client/controllers/list-clientmanagements.client.controller.js

```diff
@@ -11,19 +11,79 @@
     var vm = this;
 
     vm.clientmanagements = ClientmanagementsService.query();
-    vm.headSort = 'name';
+    vm.headSort= 'name';
```

Comment:
Some logic in this file is incorrect. Sorting any column changes the sorting of all of them